Meden-Agan-27/travel-worms
# Reviews feedback

What I did:
1. Ajaxified reviews
2. Updated timestamp
3. Separated reviews into cards with borders

How to test:
1. Log in from localhost:3000 and write a review for a book. You should immediately see it added under "Reviews" without the page refreshing, and the review form should re-load.
2. See the timestamp of the review as format "Month Day, Year"
3. See all reviews under "Reviews" separated into cards with borders.
